androidtv/basetv/basetv.py

```diff
@@ -134,17 +134,15 @@ def _cmd_audio_state(self):
         if constants.CUSTOM_AUDIO_STATE in self._custom_commands:
             return self._custom_commands[constants.CUSTOM_AUDIO_STATE]
 
-        # Is this an Android 11 device?
-        if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") == "11":
-            return constants.CMD_AUDIO_STATE11
-
-        # Is this an Android 12 device?
-        if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") == "12":
+        # Is this an Android 11-14 device?
+        if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") in [
+            "11",
+            "12",
+            "13",
+            "14",
+        ]:
```

Comment:
Question: to make this more future-proof, should the SW version be converted to an int and then check that it's `>= 11`? 

Some care would need to be taken to avoid exceptions when converting to an int.

Reply:
Looked at it a bit more, it's going to get really messy to handle those versions correctly... Right now `""` means "unknown version" and with an int it would have to be None or something which you can't directly compare...